samples/TagHelperSample.Web/Views/Shared/_Layout.cshtml

```diff
@@ -1,9 +1,9 @@
 ﻿<!DOCTYPE html>
 <html>
 <head>
-    <link href="~/blank.css" rel="stylesheet" />
-    <link href="~/site.min.css" rel="stylesheet"
-          asp-fallback-href="~/site.css"
+    <link href="~/css/sub/blank.css" rel="stylesheet" data-test="value" />
+    <link href="~/site.min.css" rel="stylesheet" data-test="value"
+          asp-fallback-href-include="**/blank.css,**/site.css"
           asp-fallback-test-class="fallback-test"
```

Comment:
More of  a question: How is globbing and href mix together. I'm assuming that if you go to a CDN you get a single file, and your test will just test against that. Why wouldn't you have a single file on the site to use with globbing? 

I can think of some scenarios where you have multiple href references to a remote/CDN, but then the globbing seems wierd
Can we share the realistic user scenario here?